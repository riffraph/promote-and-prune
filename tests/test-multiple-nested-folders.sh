#!/bin/sh

TESTDATAFOLDER=./testMultipleNestedFoldersDataset

# test of multiple nested folders

mkdir -p $TESTDATAFOLDER/folder1/folder1a
mkdir -p $TESTDATAFOLDER/folder1/folder1b
mkdir -p $TESTDATAFOLDER/folder1/folder1c

head -c 8 /dev/urandom > $TESTDATAFOLDER/folder1/folder1a/file1a
head -c 1024 /dev/urandom > $TESTDATAFOLDER/folder1/folder1a/file2a
head -c 8 /dev/urandom > $TESTDATAFOLDER/folder1/folder1b/file1b
head -c 1024 /dev/urandom > $TESTDATAFOLDER/folder1/folder1b/file2b
head -c 8 /dev/urandom > $TESTDATAFOLDER/folder1/folder1c/file1c
head -c 1024 /dev/urandom > $TESTDATAFOLDER/folder1/folder1c/file2c

identify.sh $TESTDATAFOLDER
promote.sh $TESTDATAFOLDER


# rm -rf $TESTDATAFOLDER
