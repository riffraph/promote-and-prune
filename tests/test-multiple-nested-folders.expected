-- IDENTIFY --
8 ./testMultipleNestedFoldersDataset/folder1/folder1a/file2a
8 ./testMultipleNestedFoldersDataset/folder1/folder1b/file2b
8 ./testMultipleNestedFoldersDataset/folder1/folder1c/file2c
-- PROMOTE --
TODO
